content/posts/0.20-release.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Pyodide 0.12.0 release"
+date: 2022-01-10
+draft: true
```

Comment:
I haven't really found a use for it. Probably better to always keep it to false?

Reply:
Or leave it out entirely? The `./hugo create` script adds this.